test/module.test.js

```diff
@@ -49,6 +49,18 @@ describe('basic', () => {
         rel: 'alternate',
         href: 'nuxt-app.localhost/fr',
         hreflang: 'fr-FR'
+      },
+      {
+        tagName: 'link',
+        rel: 'alternate',
+        href: 'nuxt-app.localhost/',
+        hreflang: 'en'
+      },
+      {
+        tagName: 'link',
+        rel: 'alternate',
+        href: 'nuxt-app.localhost/fr',
+        hreflang: 'fr'
```

Comment:
I'd like to also have a test where locale iso code is initially two letter locale. I believe this is still a valid ISO code and people might be using such configuration already. In that case, no extra meta tags should be added for that locale.

So configuration like:
```
locales: [{ code: 'en', iso: 'en' }, { code 'fr', iso: 'fr-FR' }]
```

Reply:
Will cover that in a test case. Should I add a new fixture or alter the current one?

Reply:
Covered it in the same fixture for now

Reply:
You don't have to create new fixture but you can override settings for the existing one. Like in `differentDomains enabled` test.